dmplugin/dmclient.go

```diff
@@ -373,13 +373,13 @@ func (dm *DataMoverClient) processActions(ctx context.Context) chan *dmAction {
 		if !ok {
 			msg := fmt.Sprintf("Received cancel for a non-existent action: %s", action.item.PrimaryPath)
 			alert.Warnf(msg)
-			action.Finish(errors.New(msg))
+			// Do not call action.Finish here, as there is no actual action in-flight to Finish.
 			return
 		}
 
 		alert.Writer().Log(fmt.Sprintf("id:%d Cancel %s", action.item.Id, action.item.PrimaryPath))
 		cancel.(context.CancelFunc)()
-		action.Finish(nil)
+		// Do not call action.Finish.  The action being canceled will call Finish with an error indicating cancelation.
```

Comment:
Isn't the lustre coordinator interested in knowing whether the actual cancel action was successful or not. I see a situation where, save for the op timeout,  going "mute" here would make the coordinator not know what happened to the cancel action.